<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plotSensorFull" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotSensorZoom" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTi">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>t_i:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTf">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>t_f:</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotSensorResampled" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotSensorRegression" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTimeBegin">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTimeEnd">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonZoom">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAcquire">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Acquire Sensor
Plot Raw</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listMaxima">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonResample">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Resampled
Find Maxima</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRegression">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>680</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute Regression</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listRegression">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>740</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTi_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>519</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Peaks of &quot;hip_angle&quot; trace</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
